tests/nfs3-readdirplus/test.yaml

```diff
@@ -0,0 +1,33 @@
+requires:
+  features:
+    - RUST
+  files:
+    - rust/src/nfs/nfs3.rs
+
+args:
+- -k none
+- --set stream.midstream=true
+
+checks:
+  - filter:
+      count: 1
+      match:
+        event_type: nfs
+        dest_ip: 192.168.1.6
```

Comment:
I see src/dst ip interchanged in 5.. I remember some work done about proper tx logging.. Maybe that's related. Perhaps we should add a min version check for 6? @victorjulien ?

Reply:
yeah lets go with min version 6